src/components/document/sort-work-view.scss

```diff
@@ -1,21 +1,145 @@
+@import "../vars";
+$title-margin: 2px;
+
 .sort-work-view {
   height: 100%;
   width: 100%;
   overflow-y: auto;
 
-  .sort-work-documents {
+  .sorted-sections{
+    width: 100%;
+    height: 100%;
+
+    .section-header{
+      width: 100%;
+      height: 30px;
+      position: relative;
+      margin-top: 5px;
+      margin-bottom: 5px;
+
+      &::after{ //divider line drawn across
+        content: "";
+        position: absolute;
+        left: 0px;
+        right: 0px;
+        bottom: 50%;
+        border-bottom: 1px solid $charcoal-light-1;
+      }
+
+      .section-header-label{
+        position: absolute;
+        left: 10px;
+        width: 71px;
+        height: 26px;
+        border-radius: 5px;
+        background-color: $classwork-purple-light-7;
+        font-weight: bold;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        color: var(--text-black);
+        z-index: 1;
+      }
+    }
+  }
+
+  .focus-document {
```

Comment:
So the CSS was basically copied over from `section-document-or-browser.scss` 
So I was thinking since the "My Work" and "Class Work" tab only use `<SectionDocumentOrBrowser>`, and since we are phasing those out (correct me if I'm wrong - I believe you mentioned Leslie saying that). I just figured I'd copy it over  - I mean I figure we'd eventually get rid of `section-document-or-browser.scss`

Reply:
We'd  also eventually want to move this CSS into a common file that all Nav Tabs that open up thumbnail documents could point to since these are children of parent classes with prefix "sort-work" etc.

Reply:
The best refactoring in the future would be to extract a component from DocumentView which just renders the document with the title. And then DocumentView can overlay its paging stuff on top of that. But that can be for another time when this has settled down.  

I'm not sure if we'll be getting rid of "My Work", but I have heard that "Class Work" will go away if the new Sort Work tab works well for users.